src/libs/actions/Report.js

```diff
@@ -271,6 +250,22 @@ function addActions(reportID, text = '', file) {
         },
     ];
 
+    const successData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
+            value: _.mapObject(optimisticReportActions, () => ({pendingAction: null})),
+        },
+    ];
+
+    const failureData = [
```

Comment:
Definitely out of scope of this PR but we should have set lastMessageText as the previous non-failure message text in `failureData`.

It would have prevented bug - https://github.com/Expensify/App/issues/19913